easybuild/easyconfigs/b/BCALM/BCALM-2.2.0-foss-2018a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'BCALM'
+version = '2.2.0'
+
+homepage = 'https://github.com/GATB/bcalm'
+description = """de Bruijn graph compaction in low memory"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/GATB/bcalm/archive']
+sources = [
+    'v%(version)s.tar.gz',
+    {
+        'source_urls': ['https://github.com/GATB/gatb-core/archive'],
+        'download_filename': 'da36e81667f46fd34aa274e792d280.tar.gz',
+        'filename': 'gatb-core-20180418.tar.gz',
+        'extract_cmd': 'tar -C bcalm-2.2.0/gatb-core --strip-components=1 -xf %s'
+    },
+]
+patches = ['BCALM-2.2.0-fix-nogit.patch']
```

Comment:
Same here, please use `%(version)s` when it matches (serves as a reminder to re-check the patch when updating the version)